front/components/app/NewBlock.tsx

```diff
@@ -166,14 +166,13 @@ export default function NewBlock({
               <div className="ml-4 grid max-w-md grid-cols-12 items-center">
                 <div className="col-span-4 sm:col-span-3">
                   <div className="flex text-base font-medium text-gray-900">
-                    {block.typeNames.map((type) => (
-                      <div
-                        key={type}
-                        className="mr-1 rounded-md bg-gray-200 px-1 py-0.5 text-sm font-bold"
-                      >
-                        {type}
-                      </div>
-                    ))}
+                    <div
+                      className={`mr-1 rounded-md px-1 py-0.5 text-sm font-bold ${
+                        block.type === "input" ? "bg-orange-200" : "bg-gray-200"
+                      }`}
+                    >
+                      {block.name}
```

Comment:
Kind of expecting @spolu to say that we still need to display the type here. Question is why? Type is likely more important in the docs that in the webapp?

Reply:
Let's keep type for now as it is i,portant that these tag match the type tag on each block :+1: 

We can revisit that later of course.

Reply:
<img width="512" alt="Screenshot 2024-04-29 at 10 35 21" src="https://github.com/dust-tt/dust/assets/1189312/7cbf2c68-7f82-4870-9931-96b186f3a0ec">
Does this look OK?